WooCommerce/Resources/Woo-Alpha.entitlements

```diff
@@ -22,6 +22,18 @@
 	</array>
 	<key>com.apple.developer.ubiquity-kvstore-identifier</key>
 	<string>$(TeamIdentifierPrefix)$(CFBundleIdentifier)</string>
+	<key>com.apple.security.app-sandbox</key>
```

Comment:
I'm not all that familiar with Catalyst – do we need the keys repeated in this entitlements file?

It's not hurting anything to have them, more just curious

Reply:
@jkmassel Thanks for pointing this out, because I hadn't looked in detail at it... 

Xcode added these entitlements when Mac Catalyst was enabled, based on what the iOS uses, as described in [this help article](https://help.apple.com/xcode/mac/current/#/dev8e94ce3c8). 

![CleanShot 2022-01-27 at 09 43 16@2x](https://user-images.githubusercontent.com/2472348/151333728-ed08c3da-96e3-4b9a-b871-94db2c9da810.png)

I'm going to leave them in since it's not hurting anything, and if we ever release a Catalyst version it'll cause pain if these aren't here. If you think anyone else should cast their eye over this bit before it's merged, please do say!